src/isPackageUpdated.js

```diff
@@ -1,6 +1,11 @@
 import getUpdatedPackages from './getUpdatedPackages';
 
 export default function isPackageUpdated(pkgName, options = {}) {
-  return !!getUpdatedPackages(options)
-    .find(pkg => pkg.name === pkgName);
+  /*
+    getUpdatedPackages returns a Buffer via `execSync`.
+
+    As of Node v5.3.0 Buffer instances get an `includes` method
+    used to search the Buffer for a value.
+ */
+ return getUpdatedPackages(options).includes(pkgName);
```

Comment:
We should be able to leave the old logic as-is. Checking that the package name is included in the buffer would lead to bugs if your repo contains packages with names that are also keys in the json output from lerna such as `name`, `version`, `private`, or `location`.